src/lib/base/secmem.h

```diff
@@ -82,7 +82,12 @@ class secure_allocator
          ::new(static_cast<void*>(p)) U(std::forward<Args>(args)...);
          }
 
+#ifdef _MSC_VER
+#pragma warning(push)
+#pragma warning(disable: 4100)
       template<typename U> void destroy(U* p) { p->~U(); }
```

Comment:
I fix this warning locally with this:
template<typename U> void destroy(U* p) { p = p; p->~U(); }

Not that it really matters.